Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -296,10 +296,11 @@ def rasterize_command():
     r_type = demisto.args().get('type', 'png')
     wait_time = int(demisto.args().get('wait_time', 0))
     page_load = int(demisto.args().get('max_page_load_time', DEFAULT_PAGE_LOAD_TIME))
+    filename = demisto.args().get('filename', 'url')
```

Comment:
Please change the argument name `filename` to `file_name` in all this file.
```suggestion
    file_name = demisto.args().get('file_name', 'url')
```